### PR TITLE
Feature/682 adjust heading levels

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [HMW-682](https://jira.epa.gov/browse/HMW-682)

## Main Changes:
* Wrapped the "How's My Waterway?" banner in an `h1` tag, and bumped other headings down. It should be the only `h1` tag, now.
* Fixed some other style issues, including a bug introduced recently that caused the Tribal page upstream watershed dialogue to not show.

## Steps To Test:
1. Test the full application for proper heading structure using the WAVE evaluation tool.

### CODE REVIEW

**Comment by maxdiebold-erg** [comment]
@cschwinderg I did find one skipped heading in the legend, because the "Past Water Conditions" legend is straight from Esri. I can address that now, or we can let that slide, what do you think?

**Comment by cschwinderg** [comment]
I don't think we should try to modify the Esri legend items. I'm ok with just letting it slide.

**Comment by cschwinderg** [comment]
Maybe we should just put an `<h2>Legend</h2>` at the top and style it similar to the headings for other widgets (i.e., basemap/layerlist and surrounding widgets).

**Comment by maxdiebold-erg** [comment]
I was leaning toward the "Legend" heading, too.

File: app/client/src/components/shared/AddSaveDataWidget.js [hunk 1]
```diff
@@ -127,18 +127,20 @@ const layerPanelStyles = (visible) => css`
...
+    padding: 7px 15px;
+    border-bottom: 1px solid #ccc;
+    color: #898989;
+    font-size: 16px;
+    font-weight: normal;
+  }
 `;
 
+const layerPanelHeaderStyles = css``;
+
```
Comment (cschwinderg) [inline app/client/src/components/shared/AddSaveDataWidget.js:143]:
```suggestion
```

This can be removed as it is no longer used.

**Review by cschwinderg — APPROVED** [review]
Looks good!

File: app/client/src/components/shared/AddSaveDataWidget.js (hunk 1 above)
Comment (cschwinderg) [reply app/client/src/components/shared/AddSaveDataWidget.js]: I went ahead and removed it and commited the change.

Comment (maxdiebold-erg) [reply app/client/src/components/shared/AddSaveDataWidget.js]: Thank you.

**Pull request merged**